sotopia/envs/evaluators.py

```diff
@@ -424,24 +427,34 @@ def unweighted_aggregate_evaluate(
     ):
         log.debug(f"[green] The conversation is terminated. {response}")
     return ScriptEnvironmentResponse(
-        terminated=environment_responses[0]["terminated"]
-        if "terminated" in environment_responses[0]
-        else False,
+        terminated=(
+            environment_responses[0]["terminated"]
+            if "terminated" in environment_responses[0]
+            else False
+        ),
         p1_rate=(
-            agent_1_responses[0]["overall_score"]
-            if "overall_score" in agent_1_responses[0]
-            else 0,
-            agent_1_responses[0],
-        )
-        if agent_1_responses != ({}, "")
-        else None,
+            (
```

Comment:
Why are we having this more complex structure here?

Reply:
oops, my formatter did this automatically. I can revert it